services/api-server/openapi.json

```diff
@@ -5544,7 +5544,7 @@
           "urls": {
             "items": {
               "type": "string",
-              "maxLength": 65536,
+              "maxLength": 2083,
```

Comment:
This look strange. Do you mind checking if it's correct? Also the ones below seem similar

Reply:
@GitHK. it is correct. Check utils_pydantic and the doc in there